osbs/cli/main.py

```diff
@@ -600,12 +603,19 @@ def cli():
                               help="perform a scratch build")
     build_parser.add_argument("--yum-proxy", action='store', required=False,
                               help="set yum proxy to repos from koji/add-yum-repo params")
-    build_parser.add_argument("--worker", action="store_true", required=False,
-                              default=False, help="create worker build")
     build_parser.add_argument("--platform", action='store', required=False,
-                              help="platform name to use")
+                              help="platform name to use (worker build)")
+    build_parser.add_argument("--platforms", action='append', metavar='PLATFORM',
+                              help="name of each platform to use (orchestrator build)")
```

Comment:
Unfortunately, `argparse` doesn't provide something like this. I created some helper functions to do so. Let me know what you think.

I also added [groups](https://docs.python.org/2.7/library/argparse.html#argument-groups) for worker and orchestrator. I found it might be useful when using `--help` .

```
$ osbs build --help
usage: osbs build ...

optional arguments:
  -h, --help            show this help message and exit
...

arguments for --worker:
  Required arguments for creating a worker build

  --platform PLATFORM   platform name to use
  --release RELEASE     release value to use

arguments for --orchestrator:
  Required arguments for creating an orchestrator build

  --platforms PLATFORM  name of each platform to use
```